src/creatures.py

```diff
@@ -140,30 +142,63 @@ def equip_weapon(self, weapon: Weapon):
     def __repr__(self):
         return f"{self._name} ({self.current_hp if Condition.down not in self.__conditions else 'down'})"
 
+    def __get_available_melee_attacks(self, grid: Grid) -> Set[MeleeAttack]:
+        creature_location = grid.find(self)
+        return {
+            MeleeAttack(self, cast(Creature, target)) for target in grid.get_adjacent_items(creature_location)
+            if issubclass(type(target), Creature)
```

Comment:
Can we not attack non-creatures? 🤔 
I.e. break a window

Reply:
I think we want to separate the logic for attacking creatures and objects - they behave a bit differently. For example, how would you treat attacking a tree? can you chop a tree by stabbing it or hitting it with a club?

Reply:
Makes sense. Resolved 👍